src/r_main.c

```diff
@@ -967,32 +967,32 @@ void R_InitAnyRes(void)
 void R_BindRenderVariables(void)
 {
   BIND_NUM_GENERAL(extra_level_brightness, 0, 0, 4, "Level brightness");
-  BIND_BOOL_GENERAL(stretchsky, false, "1 to stretch short skies");
-  BIND_BOOL_GENERAL(linearsky, false, "1 for linear horizontal sky scrolling");
-  BIND_BOOL_GENERAL(r_swirl, false, "1 to enable swirling animated flats");
-  BIND_BOOL_GENERAL(smoothlight, false, "1 to enable smooth diminishing lighting");
+  BIND_BOOL_GENERAL(stretchsky, false, "Stretch short skies");
+  BIND_BOOL_GENERAL(linearsky, false, "Linear horizontal scrolling for skies");
+  BIND_BOOL_GENERAL(r_swirl, false, "Swirling animated flats");
+  BIND_BOOL_GENERAL(smoothlight, false, "Smooth diminishing lighting");
   M_BindBool("voxels_rendering", &default_voxels_rendering, &voxels_rendering,
-             true, ss_none, wad_no, "1 to enable voxels rendering");
+             true, ss_none, wad_no, "Allow rendering of voxels");
   BIND_BOOL_GENERAL(brightmaps, false,
-    "1 to enable brightmaps for textures and sprites");
+    "Brightmaps for textures and sprites");
   BIND_NUM_GENERAL(invul_mode, INVUL_MBF, INVUL_VANILLA, INVUL_GRAY,
-    "Invulnerability effect (0 = Vanilla, 1 = MBF, 2 = Gray)");
-  BIND_BOOL(flashing_hom, true, "1 to enable flashing HOM indicator");
-  BIND_NUM(screenblocks, 10, 3, 11, "Initial play screen size");
+    "Invulnerability effect (0 = Vanilla; 1 = MBF; 2 = Gray)");
+  BIND_BOOL(flashing_hom, true, "Enable flashing of the HOM indicator");
```

Comment:
`Flashing HOM indicator`

Reply:
To me, that implies that the setting would enable the whole thing, whereas it just makes the indicator flash when enabled, which has to be done manually through the `HOM` cheat.

Reply:
Makes sense.